app/serializers/api/admin/order_serializer.rb

```diff
@@ -8,6 +11,48 @@ def full_name
     object.billing_address.nil? ? "" : ( object.billing_address.full_name || "" )
   end
 
+  def distributor_name
+    object.distributor.andand.name
+  end
+
+  def show_path
+    return '' unless object.id
+    Spree::Core::Engine.routes_url_helpers.admin_order_path(object)
```

Comment:
you could move `Spree::Core::Engine.routes_url_helpers` to a method to DRY things a bit.